composer.json

```diff
@@ -26,14 +26,15 @@
     "autoload": {
         "files": [
             "./src/Error/helpers.php",
+            "./src/Execution/values.php",
             "./src/Language/blockStringValue.php",
-            "./src/Language/helpers.php",
+            "./src/Language/utils.php",
```

Comment:
I would a void general name like `utils`. Because we have two `utils.php` file here. Even `/Util/util`.php

Reply:
Could we have `/utils/language.php` , `/utils/type.php` for example?

Reply:
What would be a better name?

Reply:
@crisu83 see my comment above : D

Reply:
We could, but on the other hand I would like the packages to be self-contained, if possible.